spec/services/preserved_object_handler_update_version_spec.rb

```diff
@@ -123,19 +123,12 @@
           context 'ActiveRecordError' do
             let(:results) do
               allow(Rails.logger).to receive(:log)
-              po = instance_double('PreservedObject')
-              allow(po).to receive(:current_version).and_return(1)
+              po = instance_double('PreservedObject', current_version: 1)
```

Comment:
any particular reason for leaving `po` as a double while switching `pc` to being an actual object from the factory?  or just didn't switch it because it doesn't make a practical difference in the test?

Reply:
mostly because it was a one liner and made no difference to the test.  I was lazy.

Reply:
totally reasonable, just had a preference for it being intentional and not an oversight =)